src/main/resources/application.yaml

```diff
@@ -9,6 +9,12 @@ zeebe:
         clusterName: dev
         connectionTimeout: PT30S
 
+# Options: hazelcast | kafka
+# This config switches importers between the provided
+# To use each of them, zeebe must be configured using hazelcast-exporter or kafka-exporter, respectively
+# See the examples in docker/docker-compose.yml in services.zeebe-hazelcast and services.zeebe-kafka
+zeebe-importer: ${ZEEBE_IMPORTER:hazelcast}
```

Comment:
Clarification question: why using this construct `${ZEEBE_IMPORTER:hazelcast}`?
IMHO spring properties automatically will pick up environment variables, there is no ned for this special syntax here, no?

Reply:
You're right, I should remove it. It's just my habit to create such variables, because it's been accepted in many projects. So that environment variables set via Vault, Helm, or other external source have the same standard

Reply:
Right, I would prefer "vanilla" Spring declarations, without the nesting approach.
E.g. `zeebe-importer: hazelcast`